src/ProtocolGateway.Core/ProtocolGateway.Core.csproj

```diff
@@ -32,6 +32,24 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x64'">
```

Comment:
what is the value of having a x64-specific configuration (and I assume compiling them as x64 only) for projects under /src? I suppose SF is x64 only but it should be fine to just reference "Any CPU" version of /src projects under SF-specific projects.